rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -1389,7 +1415,12 @@ private void processSigning(BtcECKey federatorPublicKey, List<byte[]> signatures
             try {
                 sig = BtcECKey.ECDSASignature.decodeFromDER(signatures.get(i));
             } catch (RuntimeException e) {
```

Comment:
Now, this exception is not being used either for logging the error message or rethrowing the exception. What about logging the exception message and the stack trace?